test/trento/application/projectors/host_projector_test.exs

```diff
@@ -166,6 +167,28 @@ defmodule Trento.HostProjectorTest do
                      1000
   end
 
+  test "should update the selected_checks field when event is received" do
+    %{id: host_id} = insert(:host, id: UUID.uuid4())
+
+    test_checks_data = [
+      %{checks: [Faker.StarWars.character(), Faker.StarWars.character()]},
+      %{checks: []}
+    ]
+
+    Enum.each(test_checks_data, fn %{checks: checks} ->
```

Comment:
I don't know if people is aware of the common `for` in elixir hehe
`for %{checks: checks} <- tests_checks_data do`

Reply:
Oki fair point, Enum.each was just the first thing which shot in my mind :D